neurodsp/plts/combined.py

```diff
@@ -0,0 +1,63 @@
+"""Plotting functions for plots with combined panels."""
+
+import matplotlib.pyplot as plt
+
+from neurodsp.spectral import compute_spectrum
+from neurodsp.spectral.utils import trim_spectrum
+from neurodsp.plts.spectral import plot_power_spectra
+from neurodsp.plts.time_series import plot_time_series
+from neurodsp.plts.utils import savefig
+
+###################################################################################################
+###################################################################################################
+
+@savefig
+def plot_timeseries_and_spectrum(times, sig, fs, f_range=None, spectrum_kwargs=None,
+                                 ts_kwargs=None, psd_kwargs=None, **plt_kwargs):
+    """Plot a timeseries together with it's associated power spectrum.
+
+    Parameters
+    ----------
+    times : 1d array, or None
+        Time definition(s) for the time series to be plotted.
+        If None, time series will be plotted in terms of samples instead of time.
+    sigs : 1d array
+        Time series to plot.
+    fs : float
+        Sampling rate, in Hz.
+    f_range : list of [float, float], optional
+        The frequency range to restrict the power spectrum to.
+    **spectrum_kwargs : dict, optional
+        Keyword arguments for computing the power spectrum.
+        See `compute_spectrum` for details.
+    **ts_kwargs : dict, optional
+        Keyword arguments for customizing the time series plot.
+    **psd_kwargs : dict, optional
```

Comment:
from `**psd_kwargs` up to `**spectrum_kwargs`, these aren't being capture by **kwargs, so I think the **'s can be removed since it's explicitly a dict and not an unpacked dict.

Reply:
Ooops, yeh, you're totally right - fixed!